INSTETC/configure/RELEASE

```diff
@@ -1,8 +1,26 @@
-# top level master release and local private options 
-include $(TOP)/../../../configure/MASTER_RELEASE
```

Comment:
there are quite a lot of other places that use MASTER_RELEASE - do you think we should remove them all and just put in what's needed? perhaps in another ticket as I think it's a fair bit of work?

Reply:
Yes we should, and there is already a ticket to do that but the list is quite long...   ISISComputingGroup/IBEX#8565